source/macos/AdaptiveCards/AdaptiveCards/Views/ACRContentStackView.swift

```diff
@@ -220,6 +226,19 @@ class ACRContentStackView: NSView, ACRContentHoldingViewProtocol, SelectActionHa
         layer?.backgroundColor = ColorUtils.hoverColorOnMouseEnter().cgColor
     }
     
+    func setErrorMessage(with title: String, for view: InputHandlingViewProtocol) {
+        errorMessageView = NSTextField()
+        errorMessageView?.stringValue = title
+        errorMessageView?.isHidden = true
+        errorMessageView?.isEditable = false
+        errorMessageView?.isBordered = false
+        errorMessageView?.isSelectable = true
+        errorMessageView?.backgroundColor = .clear
+        view.errorMessageHandler = self
+        guard let errorMessageView = errorMessageView else { return }
+        addArrangedSubview(errorMessageView)
```

Comment:
```suggestion
        let textfield = NSTextField()
        textfield.stringValue = title
        textfield.isHidden = true
        textfield.isEditable = false
        textfield.isBordered = false
        textfield.isSelectable = true
        textfield.backgroundColor = .clear
        view.errorMessageHandler = self
        addArrangedSubview(textfield)
        errorMessageView = textfield
```

If flipped this way, optional chaining and the guard statement can be avoided.

Reply:
Also, should we check if errorMessageView is not nil and re-use it if already created?